reference/tax_zone_check.v3.yml

```
@@ -0,0 +1,184 @@
+openapi: '3.0.0'
+info:
+  title: Tax Zone Check
+  description: Check what tax zone a given address and customer group derives to.
```

```suggestion
  description: Check what tax zone a given address and customer group derives to.
```

Determine the applicable tax zone for a given address and customer group.